Riot/SupportingFiles/Riot-Bridging-Header.h

```diff
@@ -46,3 +45,21 @@
 #import "RoomInputToolbarView.h"
 #import "NSArray+Element.h"
 #import "ShareItemSender.h"
+
+// MatrixKit common imports
+#import "MatrixKit-Bridging-Header.h"
+
+// MatrixKit application imports
+#import "MXKBarButtonItem.h"
```

Comment:
Why not adding those headers to MatrixKit-Bridging-Header.h?

Reply:
Extension targets don't need all "MXK" files. "MatrixKit-Bridging-Header.h" file is used to expose minimum MatrixKit files needed for Swift usage in all targets.

Reply:
Ok, I got it now. We should improve the comment in the code to make it clearer for the future like:
```
// MatrixKit imports for the application target (Riot)